### PR TITLE
Taltech added

### PR DESCRIPTION
**Pull request by Maxnm47**

### CODE REVIEW

**Review by askeko — CHANGES_REQUESTED** [review]
I think the comments to this pull request breaks some GDPR laws

**Comment by Maxnm47** [comment]
> I think the comments to this pull request breaks some GDPR laws

no

File: core/notebooks/00/caltech_CNN_LSTM/00_CNN_LSTM_caltech.ipynb
Comment (Tinggaard) [inline core/notebooks/00/caltech_CNN_LSTM/00_CNN_LSTM_caltech.ipynb:1]: And clear output - it looks very scuffed :)

File: core/preprocessing/acn.py [hunk 1]
```diff
@@ -129,7 +130,15 @@ def process_ev_data() -> None:
         ]
     ]
 
-    # Write the output to a CSV file
+    # Write the output to a CSV file, dont know if works but wont test 30 min no no.
```
Comment (Tinggaard) [inline core/preprocessing/acn.py:133]: what

File: core/util/get_caltech.py [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+) -> tuple[
+    torch.Tensor,
+    torch.Tensor,
+    torch.Tensor,
+    torch.Tensor,
+    torch.Tensor,
+    torch.Tensor,
+]:
+    """Divide dataset into training, validation and test sets using some block size."""
+    parks = [get_caltech_dataset(features)]
```
Comment (Tinggaard) [inline core/util/get_caltech.py:39]: There is no need for this to be a list, right?

**Comment by Tinggaard** [comment]
yes

**Comment by Tinggaard** [comment]
@Maxnm47 jeg fikser det, no worries

**Review by Tinggaard — APPROVED** [review]

**Pull request merged**